core/local/layers/events.js

```diff
@@ -0,0 +1,47 @@
+/* @flow */
+
+/*::
+import type { Metadata } from '../../metadata'
+
+export type AtomWatcherEvent = {
+  action: "created" | "modified" | "deleted" | "renamed",
+  kind: "file" | "directory" | "symlink" | "unknown",
+  path: string,
+  oldPath: string
```

Comment:
Is `oldPath` mandatory for every AtomWatcherEvent subtype?

Reply:
No, but I didn't find a way to make `flow` happy without that